superset-frontend/plugins/plugin-chart-echarts/src/utils/forecast.ts

```diff
@@ -150,3 +150,34 @@ export function rebaseForecastDatum(
     return newRow;
   });
 }
+
+// For Confidence Bands, forecast series on mixed charts require the series sent in the following sortOrder:
+export function reorderForecastSeries(row: any[]): any[] {
```

Comment:
Not sure how well-defined of a type we can have here, but it'd be nice to avoid "any"